src/AWS/Orleans.Clustering.DynamoDB/MultiClusterNetwork/GossipTableInstanceManager.cs

```diff
@@ -0,0 +1,191 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using Amazon.DynamoDBv2.Model;
+using Microsoft.Extensions.Logging;
+using Orleans.Configuration;
+using Orleans.MultiCluster;
+using Orleans.Runtime;
+using Orleans.Runtime.MultiClusterNetwork;
+
+namespace Orleans.Clustering.DynamoDB.MultiClusterNetwork
+{
+    internal class GossipTableInstanceManager
+    {
+        private const string CONF_TABLE_NAME = "OrleansGossipConfigurationTable";
+        private const string GATEWAY_TABLE_NAME = "OrleansGossipGatewayTable";
+
+        private readonly DynamoDBStorage _confStorage;
+        private readonly DynamoDBStorage _gatewayStorage;
+        private readonly string _globalServiceId;
+        private readonly ILogger _logger;
+
+        private GossipTableInstanceManager(string globalServiceId, string storageConnectionString, ILoggerFactory loggerFactory)
+        {
+            _globalServiceId = globalServiceId;
+
+            _logger = loggerFactory.CreateLogger<GossipTableInstanceManager>();
+
+            // Using legacy here
+            var options = new DynamoDBClusteringOptions();
+            LegacyDynamoDBMembershipConfigurator.ParseDataConnectionString(storageConnectionString, options);
```

Comment:
@sebastianburckhardt what is your opinion on this one ? Using Legacy* stuff makes me somehow uncomfortable, but I couldn't find a better way to do it